app/controllers/myott/subscriptions_controller.rb

```diff
@@ -2,10 +2,56 @@ module Myott
   class SubscriptionsController < ApplicationController
     before_action :disable_search_form,
                   :disable_switch_service_banner,
-                  :disable_last_updated_footnote
+                  :disable_last_updated_footnote,
+                  :cache_sections_chapters
+
+    before_action :clear_stale_flash_error, only: %i[dashboard check_your_answers]
 
     def dashboard
       @email = 'test@email.com'
+      session[:chapter_ids] = nil
+    end
+
+    def chapter_selection
+      selected_chapter_ids = Array(session[:chapter_ids])
+      all_chapters = @sections_chapters.values.flatten
+      @selected_chapters = all_chapters.select { |chapter| selected_chapter_ids.include?(chapter.to_param) }
+    end
+
+    def check_your_answers
+      selected_ids = Array(params[:chapter_ids])
+
+      if selected_ids.empty?
+        flash[:error] = 'Select the chapters you want tariff updates about.'
```

Comment:
Maybe try this so you don't need to worry about clearing the flash?
```suggestion
        flash.now[:error] = 'Select the chapters you want tariff updates about.'
```

Reply:
Yep seems to work! Thanks 🙂